docs/GettingStartedDocs/buildandsign.md

```diff
@@ -129,6 +129,16 @@ OE_SET_ENCLAVE_SGX_KSS(
     1);   /* NumTCS */
 ```
 
+On the new platform that support kss, user can also add the config_id and config_svn. The config_id can be used to contain the hash of a signing key for verifying the additional content. Similar to the relationship between MRSIGNER and ISVSVN, config_id needs a version number config_svn. After defining these properties, at running time user can make use of the additional content.
```

Comment:
```suggestion
On the new platform that supports kss, a developer can also add the config_id and config_svn. The config_id can be used to contain the hash of a signing key for verifying the additional content. Similar to the relationship between MRSIGNER and ISVSVN, config_id needs a version number config_svn. After defining these properties, at running time user can make use of the additional content.
```
Also split into multiple source lines to make github diffs easier.  (Or maybe this will go away, if I understand Simon's comment?)